### PR TITLE
ADAPT-5795 | Update Decanter and fonts; remove outdated items in html Head

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Update sans and serif fonts to Source Sans 3 and Source Serif 4 by using the latest version of Decanter 6 (v6.2.17)
- Remove outdated items in HTML head

# Review By (Date)
- Soonish?

# Criticality
- 3

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to the preview
2. Inspect any sans serif and serif text, check that it's loading the Source Sans 3 or Source Serif 4 fonts instead of the old fonts.
3. Look at things called out.

# Associated Issues and/or People
- ADAPT-5795

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,13 +49,14 @@
     "trim": "^1.0.1"
   },
   "devDependencies": {
+    "decanter": "^6.2.17",
```
Comment (yvonnetangsu) [inline package.json:52]: Updated Decanter and move it to dev dependencies

File: package.json [hunk 2]
```diff
@@ -50,13 +49,14 @@
...
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.2.1",
     "local-ssl-proxy": "^1.3.0",
     "netlify-cli": "^12.1.0",
     "node-sass": "^7.0.3",
     "node-vault": "^0.9.22",
-    "postcss": "^8.4.18",
+    "postcss": "^8.4.27",
```
Comment (yvonnetangsu) [inline package.json:59]: Update postcss as well

File: src/components/search/headerSearchButton.js [hunk 3]
```diff
@@ -5,12 +5,16 @@ const HeaderSearchButton = () => {
   const { toggleSearchOverlay } = useContext(SearchOverlayOpenContext);
 
   return (
-    <button className="ood-header__search-button" onClick={toggleSearchOverlay}>
+    <button
+      type="button"
+      className="ood-header__search-button"
+      onClick={toggleSearchOverlay}
+    >
```
Comment (yvonnetangsu) [inline src/components/search/headerSearchButton.js:12]: I meant to do this in another PR but accidentally include it here. It improves things so might as well stay here.

File: src/components/partials/htmlHead.js [hunk 4]
```diff
@@ -90,36 +90,14 @@ const HtmlHead = (props) => {
...
-        content="https://www-media.stanford.edu/assets/favicon/mstile-150x150.png"
-      />
-      <meta
-        name="msapplication-square310x310logo"
-        content="https://www-media.stanford.edu/assets/favicon/mstile-310x310.png"
-      />
-      <meta
-        name="google-site-verification"
-        content="tUJ214wF9k79KCsAn5wIOyOFR2eH0RlBANimm5MCFfU"
-      />
```
Comment (yvonnetangsu) [inline src/components/partials/htmlHead.js:119]: Removed all these outdated items in the HTML head like I did in Make a Gift

File: src/components/partials/htmlHead.js [hunk 5]
```diff
@@ -90,36 +90,18 @@ const HtmlHead = (props) => {
...
-        content="https://www-media.stanford.edu/assets/favicon/mstile-70x70.png"
-      />
-      <meta
-        name="msapplication-square150x150logo"
-        content="https://www-media.stanford.edu/assets/favicon/mstile-150x150.png"
-      />
-      <meta
-        name="msapplication-square310x310logo"
-        content="https://www-media.stanford.edu/assets/favicon/mstile-310x310.png"
-      />
```
Comment (yvonnetangsu) [inline src/components/partials/htmlHead.js:115]: Removed all these outdated items in the HTML head like I did in Make a Gift

File: gatsby-browser.js [hunk 6]
```diff
@@ -5,18 +5,18 @@
...
+export const shouldUpdateScroll = ({ routerProps: { location } }) => {
   // Prevent scrolling when user clicks on filters on Areas to Support page.
   if (location.pathname == '/areas-to-support/' && location.hash) {
     return false;
   }
   if (location.pathname.indexOf('/endowed-positions') > -1) {
     return false;
   }
-}
+};
```
Comment (yvonnetangsu) [inline gatsby-browser.js:22]: Just linter formatting fix up in this file

**Comment by yvonnetangsu** [comment]
@forrestglick FYI

File: package.json [hunk 7]
```diff
@@ -50,13 +49,14 @@
     "trim": "^1.0.1"
   },
   "devDependencies": {
+    "decanter": "^6.2.17",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.2.1",
     "local-ssl-proxy": "^1.3.0",
     "netlify-cli": "^12.1.0",
```
Comment (sherakama) [inline package.json:56]: I've been moving `netlify-cli` to `optionalDependencies` lately as it isn't even required for development.

**Comment by sherakama** [comment]
Screenshot 2023-07-27 at 14 23 06
Screenshot 2023-07-27 at 14 22 45


Looks good to me.

**Review by sherakama — APPROVED** [review]
Noice.

File: package.json (hunk 7 above)
Comment (yvonnetangsu) [reply package.json]: Totally makes sense! Just pushed up the commit 👍🏼 😄

**Comment by yvonnetangsu** [comment]
@sherakama This rebuilt on netlify ok so I'm going to merge. Somehow the status check here just doesn't sync with the latest preview deploy status
https://app.netlify.com/teams/stanford-adapt/builds/64c2e1b4202b7a0257fab835
